mllib/src/main/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluator.scala

```diff
@@ -97,6 +98,7 @@ class MulticlassClassificationEvaluator @Since("1.5.0") (@Since("1.5.0") overrid
     case "recall" => true
     case "weightedPrecision" => true
     case "weightedRecall" => true
+    case "accuracy" => true
```

Comment:
I have no strong feeling either way - but if you just set to true now, in
future you only have to add the cases that are false, and then "case _ =>
true" as the catch-all
On Wed, 4 May 2016 at 19:26, Miao Wang <notifications@github.com> wrote:

> In
> mllib/src/main/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluator.scala
> <https://github.com/apache/spark/pull/12882#discussion_r62079416>:
>
> > @@ -97,6 +98,7 @@ class MulticlassClassificationEvaluator @Since("1.5.0") (@Since("1.5.0") overrid
> >      case "recall" => true
> >      case "weightedPrecision" => true
> >      case "weightedRecall" => true
> > +    case "accuracy" => true
>
> I think one big case could be better, which leaves an incremental option
> of returning false in the future. If we simple return true now and any
> metric in the future returns false, we need to add back everything else
> returning true, which has the potential of missing some metric and is less
> obvious for people making changes.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12882/files/e7202baf1564be5ed4302c656f2abf67945ba92e#r62079416>
>